### PR TITLE
feat(dashboards): Add default widgets to library

### PR DESCRIPTION
**Pull request by shruthilayaj**
Adds 8 default widgets to the widget library. Open
to other default widgets in place of the ones suggested
here. Also, fix some padding, added cursor:pointer
and replaced border with outline to prevent css
shifts.

Screen Shot 2021-12-08 at 9 18 25 AM

### CODE REVIEW

File: static/app/views/dashboardsV2/widgetLibrary/data.tsx [hunk 1]
```diff
@@ -9,109 +9,136 @@ export type WidgetTemplate = Widget & {
...
       },
     ],
   },
   {
     id: undefined,
-    title: t('All Events'),
-    description: 'Area chart reflecting all error and transaction events.',
-    displayType: DisplayType.AREA,
+    title: t('High Throughput Transactions'),
+    description: t('Top 5 transactions with the largest volume.'),
```
Comment (narsaynorath) [inline static/app/views/dashboardsV2/widgetLibrary/data.tsx:33]: I noticed when I added this widget on prod (through dev-ui), there were 5 and then the "Other" transactions kind of shrank everything. I was just curious if we had the ability to ignore the "Other" stuff. If possible, is there a chance we might want to do that here?

Screen Shot 2021-12-08 at 1 44 42 PM

**Review by narsaynorath — COMMENTED** [review]
I just had one question, but I think these look great. The descriptions make sense and they show a range of things you can report on with the different chart types. Someone with a bit more time thinking about these might be able to provide better feedback though.

**Comment by wmak** [comment]
Hmm I like the use of `count_if` for the `slow vs fast` widget, but I wonder if this widget will be more readable since we can assign legends?
image

File: static/app/views/dashboardsV2/widgetLibrary/data.tsx (hunk 1 above)
Comment (shruthilayaj) [reply static/app/views/dashboardsV2/widgetLibrary/data.tsx:33]: It's possible to disable other by clicking on it in the legend. I kinda wish the "other" part of the graph was clickable or something so you could disable it easily...

**Comment by shruthilayaj** [comment]
> Hmm I like the use of `count_if` for the `slow vs fast` widget, but I wonder if this widget will be more readable since we can assign legends? image

yeah, the main thing I wanted to showcase with this widget was equations. Equation legends are going to be ugly for any graph type widgets, so let me see if I can shuffle things around, maybe use equations for a big number.

**Comment by shruthilayaj** [comment]
Ah, we can't do Equations on big number 😞

**Comment by shruthilayaj** [comment]
> 

big oops, forgot to add the equation part

**Review by wmak — APPROVED** [review]

**Pull request merged**